data/docmaps/regression_test/docmap_by_manuscript_id/87356.json

```diff
@@ -0,0 +1,705 @@
+                                    "Computational and Systems Biology"
+                                ],
+                                "published": "2023-06-26T14:00:00+00:00",
+                                "volumeIdentifier": "12",
+                                "electronicArticleIdentifier": "RP87356",
+                                "complement": []
+                            }
+                        }
+                    ]
+                }
+            ],
+            "assertions": [
+                {
+                    "item": {
+                        "type": "preprint",
+                        "doi": "10.7554/eLife.87356.2",
+                        "versionIdentifier": "2"
+                    },
+                    "status": "manuscript-published"
+                }
+            ],
+            "inputs": [
+                {
+                    "type": "preprint",
+                    "doi": "10.1101/2023.03.24.534142",
+                    "url": "https://www.biorxiv.org/content/10.1101/2023.03.24.534142v2",
+                    "versionIdentifier": "2"
+                },
+                {
+                    "type": "review-article",
+                    "doi": "10.7554/eLife.87356.2.sa0"
+                },
+                {
+                    "type": "review-article",
+                    "doi": "10.7554/eLife.87356.2.sa1"
+                },
+                {
+                    "type": "evaluation-summary",
+                    "doi": "10.7554/eLife.87356.2.sa2"
+                },
+                {
+                    "type": "reply",
+                    "doi": "10.7554/eLife.87356.2.sa3"
+                }
+            ],
+            "next-step": "_:b6",
+            "previous-step": "_:b4"
+        },
+        "_:b6": {
+            "actions": [
+                {
+                    "participants": [],
+                    "outputs": [
+                        {
+                            "type": "version-of-record",
+                            "identifier": "87356",
+                            "doi": "10.7554/eLife.87356.3",
+                            "versionIdentifier": "3",
+                            "published": "2024-02-14",
+                            "url": "https://doi.org/10.7554/eLife.87356.3",
+                            "content": [
+                                {
+                                    "type": "web-page",
+                                    "url": "https://elifesciences.org/articles/87356"
+                                }
+                            ]
+                        }
+                    ]
+                }
+            ],
+            "assertions": [
+                {
+                    "item": {
+                        "type": "version-of-record",
+                        "doi": "10.7554/eLife.87356.3",
+                        "versionIdentifier": "3"
+                    },
+                    "status": "vor-published"
```

Comment:
Should the status be [`version-of-record`](https://sparontologies.github.io/pso/current/pso.html#d4e992) for the first version of record version?

Reply:
While a little off topic for the current change, the EPP have discussed the need to adjust the assertion status here too. For now, we just ignore it as `vor-published` isn't valid.

I agree that possibly two assertions to replace this one: `version-of-record` and `published`. I also assume both apply to the version of record item, not applying `version-of-record` assertion to the last RPP?

Reply:
Once it is decided to use we can first update this part (either version-of-record or published) in a separate PR.

Reply:
Sounds good to me - thanks. 

>  I also assume both apply to the version of record item, not applying `version-of-record` assertion to the last RPP?

yeah I agree - thanks!

Reply:
I have updated the status for the first published version as `published` and the others as `corrected` instead of using `version-of-record `for all, since all versions already have this type. I thought this would distinguish the status and what happened more clearly. 

Please let me know if having `version-of-record` for all would be better from your perspective.

Reply:
Makes sense - thanks LGTM!

Reply:
(I needed to google LGTM :D )